src/main/markdown/faq.md

```diff
@@ -0,0 +1,49 @@
+
+If you prefer not to run a Kafka cluster, however, the Message Broker API is completely optional. You can write Lagom services that communicate with each other directly by using HTTP requests or by streaming data over a WebSocket.
+
+The Message Broker API was designed to make it possible to support for alternative message broker services in a future version of Lagom.
+
+See more in the [Java documentation](/documentation/current/java/MessageBroker.html) or the [Scala documentation](/documentation/current/scala/MessageBroker.html).
+
+## Does Lagom require using event sourcing and CQRS?
+
+You can handle data persistence any way you like in a Lagom service, but we strongly encourage event sourcing and CQRS, and the Lagom Persistence API is designed to support these patterns. For services where you do not wish to use event sourcing and CQRS, you can execute arbitrary database statements using Lagom's `CassandraSession`, `JdbcSession` or `JpaSession`. You can also use any other data access client you choose, as long as you are careful to avoid I/O blocking the primary thread pool.
+
+See more in the Lagom documentation on [Managing data persistence](/documentation/current/java/ES_CQRS.html) and [Advantages of Event Sourcing](/documentation/current/java/ESAdvantage.html).
+
+## Can I serialization methods other than JSON?
+
+Yes, Lagom supports extensible serialization, both for messages transmitted between different services and for internal messages and persistent data used within a service. JSON is supported by default, as it is simple and well understood. Services that require higher performance or more compact data can use alternatives such as [Protocol Buffers](https://developers.google.com/protocol-buffers/), [Apache Avro](http://avro.apache.org/), [Kryo](https://github.com/EsotericSoftware/kryo), or any other serializer that you choose.
+
+See more in the Lagom documentation on Message Serializers ([Java](/documentation/current/java/MessageSerializers.html)/[Scala](/documentation/current/scala/MessageSerializers.html)) and Persistent Entity Serialization ([Java](/documentation/current/java/Serialization.html)/[Scala](/documentation/current/scala/Serialization.html)).
+
+## In the example applications, why are data classes duplicated between the service API and implementation projects?
+
+This is an example of an anti-corruption layer. In Lagom, we use anti-corruption layers for all API payloads (requests/responses) and published events. While it might seem like unnecessary duplication, using anti-corruption layers allows the internal implementation to be changed without changing the public API. For example, you might want to enrich an internal event with some fields that you need on a read side but you don’t need on the published events.
+
+For more information, see Mark Needham's blog on [anti-corruption layers](http://www.markhneedham.com/blog/2009/07/07/domain-driven-design-anti-corruption-layer/).
```

Comment:
Mark Needham's post is great, I think we could add another link to https://docs.microsoft.com/en-us/azure/architecture/patterns/anti-corruption-layer too. MS's link is part of a collection of patterns so users get more for the same price.

Reply:
Upon review, this question looks a bit out of place. All other FAQ questions are common for someone who has only been exposed to Lagom for few minutes. 

If we keep this question here I think we should link to the sample applications.

Reply:
Yeah, I know what you mean. I went back and forth about whether to take this question out for this reason. It is far more detailed than the others, and maybe it is better to explore the subject elsewhere. I'll remove it.